src/snowcli/cli/snowpark/function/commands.py

```diff
@@ -163,22 +104,30 @@ def function_update(
     replace: bool = ReplaceOption,
     **options,
 ) -> CommandResult:
-    """Updates an existing python UDF or UDTF using a local artifact."""
+    """Deploy a python UDF or UDTF using a local artifact."""
     snowpark_package(
         pypi_download,  # type: ignore[arg-type]
         check_anaconda_for_pypi_deps,
         package_native_libraries,  # type: ignore[arg-type]
     )
 
     fm = FunctionManager()
-    sm = StageManager()
+    function_exists = True
+    create_or_replace = False
```

Comment:
Why is is different from `replace`?

Reply:
`create_or_replace` tells if we need to execute query

`replace` is a protection against unconscious function/procedure update